api/server.go

```diff
@@ -0,0 +1,44 @@
+package api
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/rs/zerolog/log"
+	"github.com/sprintertech/sprinter-signing/api/handlers"
+)
+
+func Serve(
+	ctx context.Context,
+	addr string,
+	signingHandler *handlers.SigningHandler,
+	statusHandler *handlers.StatusHandler,
+) {
+	r := mux.NewRouter()
+	r.HandleFunc("POST /v1/chains/{chainId:[0-9]+}/signatures", signingHandler.HandleSigning)
+	r.HandleFunc("GET /v1/chains/{chainId:[0-9]+}/signatures/{depositId}", statusHandler.HandleRequest)
+	http.Handle("/", r)
+
+	server := &http.Server{
+		Addr: addr,
```

Comment:
read timeout^^

Reply:
Will be handled when the signing api PR is merged.